chapter_optimization/optimization-intro_vn.md

```diff
@@ -17,12 +17,11 @@ If we ever need to maximize an objective there is a simple solution: just flip t
 -->
 
 Trong phần này, ta sẽ thảo luận mối quan hệ giữa tối ưu và học sâu, cũng như những thách thức khi áp dụng các thuật toán tối ưu trong học sâu.
-Đối với một bài toán trong học sâu, chúng ta thường định nghĩa hàm mất mát đầu tiên.
-Sau đó ta có thể sử dụng một thuật toán tối ưu nào đó nhằm cực tiểu hoá hàm mất mát đó.
+Đối với một bài toán học sâu, đầu tiên chúng ta thường định nghĩa hàm mất mát, 
+sau đó có thể sử dụng một thuật toán tối ưu nhằm cực tiểu hoá hàm mất mát đó.
 Trong tối ưu, hàm mất mát thường được xem là hàm mục tiêu của bài toán tối ưu.
```

Comment:
```suggestion
Trong tối ưu, hàm mất mát thường được gọi là hàm mục tiêu của bài toán tối ưu.
```

Reply:
ủa `hàm mất mát` là trong DL mà?
```suggestion
Trong học sâu, hàm mất mát được xem như hàm mục tiêu trong bài toán tối ưu.
```

Reply:
đâu, mình thấy vẫn đúng mà: "hàm mất mát trong DL khi bê sang tối ưu thì được gọi là hàm mục tiêu"

Reply:
ơ đâu nếu giữ nguyên thì là `hàm mất mát trong tối ưu` chứ?
`Trong tối ưu, hàm mất mát thường được xem là hàm mục tiêu của bài toán tối ưu` = `hàm mất mát trong tối ưu thường được xem là hàm mục tiêu của bài toán tối ưu`

Reply:
cái phần `trong DL` là imply từ mấy câu trước rồi.
tại từ `tối ưu` trong câu bị lặp 2 lần nên mới thấy rối rối :)))
`hàm mất mát "trong DL" trong context tối ưu được xem là hàm mục tiêu của bài toán tối ưu`

Reply:
oh ra vậy =))) nếu vậy `được xem là` tốt hơn `được gọi là` chứ?

Reply:
```suggestion
Hàm mất mát trong học sâu thường được xem là hàm mục tiêu của bài toán tối ưu.
```
Vậy mình viết rõ lại luôn.